test/vector-tile.test.js

```diff
@@ -2323,10 +2323,10 @@ describe('mapnik.VectorTile ', function() {
                 var validityReport3 = vtile.reportGeometryValidity({lat_lon:true});
                 var validityReport4 = vtile.reportGeometryValidity({web_merc:true});
                 assert.equal(simplicityReport.length, 0);
-                assert.equal(validityReport.length, 21); // Dataset not expected to be OGC valid
-                assert.equal(validityReport2.length, 14); // Dataset not expected to be OGC valid
-                assert.equal(validityReport3.length, 21); // Dataset not expected to be OGC valid
-                assert.equal(validityReport4.length, 20); // Dataset not expected to be OGC valid
+                assert.equal(validityReport.length, 25); // Dataset not expected to be OGC valid
+                assert.equal(validityReport2.length, 0); // Dataset not expected to be OGC valid
+                assert.equal(validityReport3.length, 26); // Dataset not expected to be OGC valid
+                assert.equal(validityReport4.length, 25); // Dataset not expected to be OGC valid
```

Comment:
@springmeyer they are completely different outputs, some of this is due to snap rounding, but in general the number of errors and warnings that boost geometry is valid reports here has completely changed. We have found that in the multipolygon case it is often wrong, this is why this has the not expected to be OGC valid is left there still. This is an old warning but none the less a problem that we have seen. Hopefully it will be fixed soon in some bug fixes that boost geometry is working on.